index.html

```diff
@@ -1122,11 +1129,290 @@ <h5>
+        </h4>
+        <p>
+          Non-government-issued credentials include all other digital
+          documents, certificates, and attestations that are not
+          government-issued and don't represent government-issued documents.
+          This could include proof of employment, (non-government) education
+          credentials, or cinema tickets. Notably, their exchange is likely
+          less restricted by laws and regulations. While these documents often
+          don't exhibit the same risks as government-issued credentials, they
+          could also contain identifiable or sensitive information.
+        </p>
+        <h5>
+          Risk of theft and leakage of non-government credentials
+        </h5>
+        <p>
+          The impact and viability of credential theft and leakage of
+          non-government credentials is largely based on the content of each
+          individual credential type. In general, it could lead to loss of
+          control and exposure of sensitive private information, as well as
+          impersonation and data theft, which can increase the likelihood of
+          further attacks on the affected individual.
+        </p>
+        <h5>
+          Risk of proliferation of requests for government credentials
+        </h5>
+        <p>
+          The flexibility and lack of regulation of non-government credentials
+          carries potential for abuse for the purpose of cross-site tracking
+          and linking identities through long-lived identifiers, such as email
+          address or phone number. Verifiers participating in a tracking scheme
+          based on digital credentials could create user incentives to accept
+          sharing identifier credentials across many sites ("loyalty cards" for
+          the Web), without fully understanding the implications on their
+          privacy.
+        </p>
+        <p>
+          Even users unwilling to share their information in such a scheme
+          could be affected by prompt fatigue and potentially risk exclusion
+          from using these services.
+        </p>
+        <h5>
+          Mitigating unnecessary requests for non-government credentials
+        </h5>
+        <p>
+          For non-government-issued credentials, it is recommended that the
+          user agent understand the requested credential format and its privacy
+          attributes, and build a risk framework that informs the context that
+          is shown to the user, as well as the amount of friction that is
+          appropriate for each credential type. Protocols and formats involved
+          in the exchange of these credentials are generally expected to
+          support features such as selective disclosure and unlinkability, but
+          these features might not always be appropriate or necessary in the
+          exchange of information, especially when it concerns low-risk
+          credentials such as cinema tickets.
```

Comment:
```suggestion
          credentials such as cinema tickets. In these cases, User Agents
          should consider the potential for fingerprinting risks if the user
          accepts the request.
```

Reply:
Yes, I agree, but we have a section on fingerprinting coming up in #283 and I'd prefer not to mix sections too much if we can avoid it :)